client/my-sites/plans-features-main/index.jsx

```diff
@@ -268,16 +268,20 @@ class PlansFeaturesMain extends Component {
 	}
 
 	render() {
-		const { site } = this.props;
+		const { site, isInSignup } = this.props;
```

Comment:
Could we explicitly add a prop for this? ie something like `displayFAQ` or `showFAQ`. I'd like to avoid tangling the `isInSignup` props to a number of side effects.